projects/scion/microfrontend-platform/src/lib/microfrontend-platform.ts

```diff
@@ -292,13 +288,13 @@ export class MicrofrontendPlatform {
     );
 
     /**
-     * Registers initializers to run in runlevel 1.
+     * Registers initializers to run in runlevel 0.
      */
-    function registerRunlevel1Initializers(): void {
+    function registerRunlevel0Initializers(): void {
       // Wait until connected to the message broker, or reject if the maximal broker discovery timeout has elapsed.
       Beans.registerInitializer({
         useFunction: () => Beans.get(BrokerGateway).whenConnected(),
-        runlevel: Runlevel.One,
+        runlevel: Runlevel.Zero,
```

Comment:
I intentionally changed connecting to the broker from runlevel 1 to runlevel 0 because we don't have other runlevel 0 logic in the client. In the host, on the other hand, we have to wait until the broker is ready and manifests have been loaded (runlevel 0), only then the host app can connect to the broker (runlevel 1).

Does this make sense? I mean, it does not really matter whether we connect in runlevel 0 or runlevel 1 from the client to the broker.

Reply:
OK, that makes sense to me.